content/kotlin/concepts/datetime/terms/toDateTimePeriod/toDateTimePeriod.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.toDateTimePeriod()'
+Description: 'Interprets an ISO-8601 formatted duration string, converting it into a DateTimePeriod object. In cases where the string's time components are either missing or amount to zero, the function yields a DatePeriod instead.'
+Subjects:
+  - 'Computer Science'
+  - 'Software Engineering'
+  - 'Other comparable degrees'
```

Comment:
Add the subjects here which are only present [here](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md)

Reply:
Thank you for this! I thought it was somewhere, did not know where to find it.

Reply:
It's okay @DylantheChicken ✅